index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title></title>
```

Comment:
## [OPTIONAL]

- [ ] Your HTML page should have a descriptive title and not be empty. I kindly suggest adding it.